### PR TITLE
rework advertise cta + add pdf download link

### PR DESCRIPTION
**Pull request by morekid**
@davidmytton added link to add kit from `/advertise/`.

Not sure where to put the .pdf file, suggestions?

### CODE REVIEW

**Comment by davidmytton** [comment]
We can link it to a Google Drive URL.

The text underneath the "download" link seems weirdly placed off to the left. Is that intended?

image

File: content/about.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ isPage: about
 headerType: fixed
 ---
 
-Discover the best tools and beta releases for developers.
-{.description}
+Discover the best tools and beta releases for developers. {.description}
```
Comment (davidmytton) [inline content/about.md:10]: Looks like an autoformat error moving `{.description}` to this line? Should be below.

File: content/about.md [hunk 2]
```diff
@@ -61,7 +60,7 @@ the best, most interesting set of tools each week:
 - Does it have good documentation? Is it fast?
 - Does it have any negative impact on security or privacy?
 - How would I feel recommending this to my friends? Would I tweet a link to it?
-{.col-2-css}
+  {.col-2-css}
```
Comment (davidmytton) [inline content/about.md:63]: This spacing breaks the custom class.

File: themes/console-home/layouts/shortcodes/br.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<br>
```
Comment (davidmytton) [inline themes/console-home/layouts/shortcodes/br.html:1]: Should this be `<br />` ?

**Comment by morekid** [comment]
@davidmytton all fixed, thanks for catching these, my bad.

The button positioning and sizing looks like a css caching issue perhaps? This is how it should look:

image

**Comment by davidmytton** [comment]
All looks good now. I'll drop in the Google Drive link later today when we have the final version.

**Pull request merged**